stylus/cozy-ui/defaults.styl

```diff
@@ -32,7 +32,20 @@ textarea
 
 // UTILS
 @import '../ui-base/mixins'
+@import '../ui-base/palette'
 :global
+    .coz-error
+        color  pomegranate
+
+        &.coz-error--warning:before
```

Comment:
hahaha 😆 
There's 2 philosophy conflicting here. Tachyons has every classes globally extended, classes with very low purposes each, but you have to extend every classes to be effective which mean the app CSS weights more than it should… always.
Or a library of extendable classes that you use when you need them which means flexibility, just the weight of CSS you need, but a more complex use.
The goal i had in mind was kinda meet them in the middle.